opendevin/agent.py

```diff
@@ -77,28 +34,8 @@ def complete(self) -> bool:
         """
         return self._complete
 
-    @property
-    def history(self) -> List[Message]:
-        """
-        Provides the history of interactions or state changes since the instruction was initiated.
-
-        Returns:
-        - history (List[str]): A list of strings representing the history.
-        """
-        return self._history
-
-    @abstractmethod
-    def add_event(self, event: Event) -> None:
-        """
-        Adds an event to the agent's history.
-
-        Parameters:
-        - event (Event): The event to add to the history.
-        """
-        pass
-
```

Comment:
Same here -- `history` is not used. Since now every `Action` and `Observation` are handled by the `Controller`, it probably make more sense to handle these histories in the `Controller` rather than in `Agent` class.